tests/rptest/scale_tests/node_operations_fuzzy_test.py

```diff
@@ -164,8 +171,8 @@ def test_node_operations(self, enable_failures, num_to_upgrade,
         if fi:
             fi.stop()
 
-        admin_fuzz.wait(20, 180)
-        admin_fuzz.stop()
+        self.admin_fuzz.wait(20, 180)
+
```

Comment:
Let's keep things explicit, if somebody copies this test without noticing tearDown they won't be aware that they should stop the fuzzer.

```
self.admin_fuzz.stop()
```

Reply:
the comment is still relevant